modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,16 @@
+def mod_expo(_base, _pow, _mod):
```

Comment:
Please make the name as `modular_exponential` . 
`mod` was OK here but `expo` isn't much of a common term for the word `exponential` / `exponent`.